lib/vdsm/virt/backup.py

```diff
@@ -667,7 +673,13 @@ def _create_scratch_disks(vm, dom, backup_id, backup_disks):
         except Exception:
             _remove_scratch_disks(vm, backup_id)
             raise
-        backup_disk.scratch_disk = ScratchDiskConfig(path=path, type="file")
+        backup_disk.scratch_disk = ScratchDiskConfig(
+            # TODO: what should be the values of these 3 fields?
+            img_id=None,
+            dom_id=None,
+            vol_id=None,
```

Comment:
Since this scratch disk is not a transient disk and not an image, they don't have 
the ids. No change is needed in this code - if you do't specify the ids, using;

    self.name = kw.get("name")

Will initialize the instance variables to None.

Not having ids is not an issue for the code adding the scratch disk ids
to the the scratch_disk dict in the drive, since we don't keep file based
scratch disks.